src/main/webapp/app/tasks/local-task.service.spec.ts

```diff
@@ -52,4 +53,21 @@ describe('LocalTaskService', () => {
     expect(localStorage.getItem).toHaveBeenCalled();
     expect(localStorage.setItem).toHaveBeenCalled();
   });
+
+  it('should update the status of the task', () => {
+    const data = {id: 'db1c4afe-d358-4b14-acfd-0953e882a6b6', name: 'Test Task', isCompleted: false};
+    // when
+    taskService.update(id, data);
+
+    // then
+    expect(data).toEqual({id: 'db1c4afe-d358-4b14-acfd-0953e882a6b6', name: 'Test Task', isCompleted: true});
```

Comment:
you should rather test the return of the service method

Reply:
Thank you 👍